stream_alert_cli/config.py

```diff
@@ -113,6 +113,10 @@ def set_prefix(self, prefix):
             LOGGER_CLI.error('Invalid prefix type, must be string')
             return
 
+        if prefix.startswith('_'):
```

Comment:
I think the prefix can't contain an underscore at all, not just start with it.

Reply:
This is correct.  an underscore in the middle of the prefix will also result in an issue in resource creation.  see https://github.com/airbnb/streamalert/issues/424.

Reply:
Ok! I'll make that change now